scripts/junos/config_tables/ConfigTables_old.yml

```diff
@@ -0,0 +1,24 @@
+# To understand the following definitions, refer to
```

Comment:
I guess you wanted to get rid of this file since you don't use it anywhere in the code and have an another almost exactly same file `ConfigTables.yml`